index.js

```diff
@@ -50,3 +66,14 @@ exports.createPassportOptions = function createPassportOptions(options) {
     deserializeNPMToken
   };
 };
+
+//
+// Timing attack safe equals for strings
+//
+function compare(a, b) {
+  var mismatch = 0;
+  for (var i = 0; i < a.length; ++i) {
```

Comment:
should this be `i++` ?

Reply:
in this specific context it behaves the same since the incrementing doesn't happen until after the first iteration with the initial value of `0`